doc/barclamp.md

```diff
@@ -1356,3 +1356,123 @@ place) you will have to add additional setup code to
 and/or
 [qa_crowbarsetup.sh](https://github.com/SUSE-Cloud/automation/blob/master/scripts/qa_crobarsetup.sh)
 to take care of your barclamp's needs.
+
+## High Availability (HA) Integration
+
+[This commit](https://github.com/crowbar/crowbar-openstack/pull/512/files) has
+more or less everything that makes a barclamp HA compatible. So we can just base
+our understanding around that commit.
+
+-  [default.rb](https://github.com/crowbar/crowbar-openstack/pull/512/files#diff-f9a1e9d161032b6c0015336ab6812c14)
+
+   This file defines the default for all the values of the vars required by barclamp.
+
+```ruby
+default[:barbican][:ha]
+# port for ha listner.
+	[:ports][:api]
+ # check interval for services
+	[:api][:op][:monitor][:interval] = "10s"
+	[:worker][:op][:monitor][:interval] = "10s"
+	[:keystone_listener][:op][:monitor][:interval] = "10s"
+	
+ # pacemaker resource agent. (SearchEngine:Pacemaker lsb)
+	[:api][:agent] = "lsb:openstack-barbican-api"
+	[:worker][:agent] = "lsb:openstack-barbican-worker"
+	[:keystone_listener][:agent] = "lsb:openstack-barbican-keystone-listener"
+
+```
+> - Resource Agent: In 'LayMan' terms, resource agent is a script that the
+    cluster manager uses to manage the resource.
+> - LSB: is a specification of writing resource agent. 
+> - OCF: Pacemaker specific extension to LSB.
+
+- [barbican_service.rb(cookbook)](https://github.com/crowbar/crowbar-openstack/pull/512/files#diff-3f95710598349c27e4d1e79ba4cd7826)
+
+   Makes the installation of the service HA aware, by using [```provider  Chef::Provider::CrowbarPacemakerService```](https://github.com/crowbar/crowbar-ha/blob/master/chef/cookbooks/crowbar-pacemaker/providers/service.rb)
```

Comment:
Please do not just link the commented code here. You can provide the link for reference, but try to explain that resource and why we need it rather than a normal service resource in the HA case.